### DISCUSSION TITLE
Support Needed for Verifying Educational Status for JetBrains Products

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142963] [GitHub Education]
### Hi everyone,

I would like to continue using JetBrains products through my educational status. Last year, I was able to verify my eligibility by submitting an official document, as my university email address cannot receive external messages. This process worked seamlessly, and I successfully activated my licenses.

This year, however, verification is managed through GitHub, which requires my university email address to be verified as my primary email on GitHub. Additionally, the verification process requires proximity to campus, which is challenging for me due to the nature of my distance learning program, as I am unable to travel to campus regularly.

As mentioned, my university email address cannot receive messages from outside the network. I am therefore unsure how to complete the verification process under these conditions.

Could you please offer an alternative solution? I would greatly appreciate your assistance in maintaining my access to JetBrains' educational resources.

**Thank you very much** for your help and support.

**Comment by ghost** [comment]
Hi, i have exactly the same issue and would like to use JetBrains products.

**Comment by DhareshP** [comment]


**Reply by ghost** [reply]
Hi Dharesh,
I don't want to offend you, but did you even read my post? The point is exactly that I can't verify my university email. The reason for this is that messages from outside the online university campus are not arriving.
